src/Form/InterviewQuestionType.php

```diff
@@ -0,0 +1,50 @@
+<?php
+
+declare(strict_types=1);
+
+namespace App\Form;
+
+use App\Dto\InterviewQuestionRequest;
+use Symfony\Component\Form\AbstractType;
+use Symfony\Component\Form\DataMapperInterface;
+use Symfony\Component\Form\Extension\Core\Type\TextType;
+use Symfony\Component\Form\FormBuilderInterface;
+use Symfony\Component\OptionsResolver\OptionsResolver;
+
+final class InterviewQuestionType extends AbstractType implements DataMapperInterface
+{
+    public function buildForm(FormBuilderInterface $builder, array $options): void
+    {
+        $builder
+            ->add('question', TextType::class)
+            ->setDataMapper($this);
+    }
+
+    public function mapDataToForms($data, $forms): void
+    {
+        if (null !== $data) {
+            $forms = iterator_to_array($forms);
+            $forms['question']->setData($data->question ?? null);
+        }
+    }
+
+    public function mapFormsToData($forms, &$data): void
+    {
+        $forms = iterator_to_array($forms);
+
+        if (null === $data) {
```

Comment:
This assertion is useless as you make the same `new InterviewQuestionRequest()` just after